obal/data/playbooks/nightly.yml

```diff
@@ -3,9 +3,28 @@
     - packages
   serial: 1
   gather_facts: no
-  roles:
-    - build_package
   vars:
-    releasers:
-      - "{{ nightly_releaser }}"
-    build_package_tito_releaser_args: "{{ nightly_package_tito_releaser_args }}"
+    build_package_tito_releaser_args: "{{ nightly_package_tito_releaser_args | default([]) }}"
+  tasks:
+    - name: 'set nightly_releaser'
+      set_fact:
+        releasers:
+          - "{{ nightly_releaser }}"
+      when: nightly_releaser is defined and "--arg jenkins_job=" in (build_package_tito_releaser_args | join(" "))
+
+    - block:
+      - name: 'set build_package_tito_releaser_args'
+        set_fact:
+          build_package_tito_releaser_args:
+            - "{{ build_package_tito_releaser_args }} --arg rpmbuild_options=\"--define 'prerelease .{{ lookup('pipe', 'date +%Y%m%d%H%M%S') }}git{{ nightly_githash[:7] }}'\""
+
+      - name: copy source files
+        copy:
+          src: "{{ item }}"
+          dest: "{{ inventory_dir }}/{{ package_base_dir }}/{{ inventory_hostname }}/"
+        with_items: "{{ nightly_sourcefiles.split(',') }}"
```

Comment:
Could `nightly_sourcefiles` be an array ? And are the sourcefiles expected to be full paths or relative?

Reply:
You know, I only tested it with absolute paths. I'm not sure about relative.

Reply:
it wants paths to be relative to the playbook which is too complicated to worry about. i'm going to say *no, it doesn't support relative paths* haha